src/allencell_ml_segmenter/_tests/fakes/fake_viewer.py

```diff
@@ -24,9 +24,10 @@ def __init__(self, viewer: Optional[napari.Viewer] = None):
         self._shapes_layers: Dict[str, ShapesLayer] = {}
         self._labels_layers: Dict[str, LabelsLayer] = {}
         self._on_layers_change_fns: List[Callable] = []
+        self.segmentation_inserted: Dict[str, np.ndarray] = {}
```

Comment:
This name is a bit misleading--this is only used for thresholds, not the initial segmentations, right? Maybe better to call it `_thresholds_inserted` or something